libs/lein-template/resources/leiningen/new/kit/README.md

```diff
@@ -17,3 +17,17 @@ Reload changes:
 ```clojure
 (reset)
 ```
+
+## REPLs
+
+### Cursive
+
+Configure a [REPL following the Cursive documentation](https://cursive-ide.com/userguide/repl.html). Using the default "Run with IntelliJ project classpath" will let you select your profiles from the ["Clojure deps" aliases selection](https://cursive-ide.com/userguide/deps.html#refreshing-deps-dependencies)
+
+### CIDER
+
+Use the profile `cider` for CIDER nREPL support. See the [CIDER docs](https://docs.cider.mx/cider/basics/up_and_running.html) for more help.
```

Comment:
I think here you mean alias, not profile?

Reply:
Good catch, alias not profile!

And sounds good, will wait your PR 👍